core/templates/pages/exploration-editor-page/modal-templates/state-version-history-modal.component.ts

```diff
@@ -51,6 +51,7 @@ export class StateVersionHistoryModalComponent
   @Input() oldState!: State;
   @Input() newStateName!: string;
   @Input() oldStateName!: string;
+  validationError: boolean = false;
```

Comment:
```suggestion
  validationErrorIsShown: boolean = false;
```

Ditto elsewhere in this file.

Reply:
Done.